source/base/static/base/js/crawl.js

```diff
@@ -30,8 +30,9 @@ $( document ).ready(function() {
   }
 
   function onOffStatus(status){
-    onOffGroup(statuses.states[status]["disabled"], statuses.buttons, true);
```

Comment:
@brittainhard - Please take a look at this, I need to understand how to reenable these.